src/cache/cache_las.c

```diff
@@ -692,6 +691,28 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
          */
         WT_ASSERT(session, __wt_count_birthmarks(first_upd) == 0);
 
+        /*
```

Comment:
I updated the comment, it says the following now:
        /*
         * If the page being lookaside evicted has previous lookaside content associated with it,    
         * there is a possibility that the update written to the disk is external (i.e. it came from 
         * the existing lookaside content and not from in-memory updates). Since this update won't   
         * be a part of the saved update list, we need to write a birthmark for it separate to       
         * processing of the saved updates.
         */